pkgs/servers/web-apps/pict-rs/default.nix

```diff
@@ -30,7 +30,7 @@ rustPlatform.buildRustPackage rec {
   PROTOC_INCLUDE = "${protobuf}/include";
 
   nativeBuildInputs = [ makeWrapper ];
-  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks ];
```

Comment:
I don't think this worked before, though I'm not sure why CI didn't catch it. There's nothing in the `checkDependencyList` logic to flatten nested lists, much less attrsets.